_templates/api-story-card/with-prompt/src/state/api.ejs.t

```diff
@@ -0,0 +1,20 @@
+---
+to: packages/<%=package%>/src/state/<%=slug%>-data/api.js
+---
+import Reduxful from "reduxful";
+import requestAdapter from "../request-adapter";
+
+const apiConfig = { requestAdapter };
+
+// FIXME: getRidershipOverTime should be variable (or more clearly mock data)
```

Comment:
I wasn't sure if we wanted to just always use this as the dummy data, but if so, would it make sense to name it something more obviously fake? Maybe something like: `getMockRidershipData`?